Error: Cannot find module 'fs/promises'

## Issue

When installing `@react-native-firebase/app` via `expo install @react-native-firebase/app`, it throws the following error:

`Error: Cannot find module 'fs/promises'`

Steps to reproduce:

- `expo init` (blank Typescript)
- `expo install expo-dev-client`
- `expo run:ios` (I did an intermediary test)
- `expo install @react-native-firebase/app @react-native-firebase/messaging @react-native-firebase/auth @react-native-firebase/perf @react-native-firebase/crashlytics`

Output:

```
Cannot find module 'fs/promises'
Require stack:
- /Volumes/ExternalSSD/Work/WeijlandIT/Expo Firebase Starter/expo-firebase-starter/node_modules/@react-native-firebase/app/plugin/build/android/copyGoogleServices.js
- /Volumes/ExternalSSD/Work/WeijlandIT/Expo Firebase Starter/expo-firebase-starter/node_modules/@react-native-firebase/app/plugin/build/android/index.js
- /Volumes/ExternalSSD/Work/WeijlandIT/Expo Firebase Starter/expo-firebase-starter/node_modules/@react-native-firebase/app/plugin/build/index.js
- /Volumes/ExternalSSD/Work/WeijlandIT/Expo Firebase Starter/expo-firebase-starter/node_modules/@react-native-firebase/app/app.plugin.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config-plugins/build/plugins/withStaticPlugin.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config-plugins/build/plugins/withPlugins.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config-plugins/build/index.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config/build/plugins/withConfigPlugins.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config/build/Config.js
- /usr/local/lib/node_modules/expo-cli/node_modules/@expo/config/build/index.js
- /usr/local/lib/node_modules/expo-cli/build/commands/installAsync.js
- /usr/local/lib/node_modules/expo-cli/build/commands/install.js
- /usr/local/lib/node_modules/expo-cli/build/commands/index.js
- /usr/local/lib/node_modules/expo-cli/build/exp.js
- /usr/local/lib/node_modules/expo-cli/bin/expo.js
```

---

## Project Files

Expo default. Did not change them manually.

### Javascript

#### `package.json`:

```json
"scripts": {
    "start": "react-native start",
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "web": "expo start --web",
    "eject": "expo eject"
  },
  "dependencies": {
    "@react-native-firebase/app": "^12.4.0",
    "@react-native-firebase/auth": "^12.4.0",
    "@react-native-firebase/crashlytics": "^12.4.0",
    "@react-native-firebase/messaging": "^12.4.0",
    "@react-native-firebase/perf": "^12.4.0",
    "expo": "~42.0.1",
    "expo-dev-client": "^0.4.7",
    "expo-splash-screen": "~0.11.2",
    "expo-status-bar": "~1.0.4",
    "expo-updates": "~0.8.1",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-native": "~0.63.4",
    "react-native-gesture-handler": "~1.10.2",
    "react-native-reanimated": "~2.2.0",
    "react-native-screens": "~3.4.0",
    "react-native-unimodules": "~0.14.5",
    "react-native-web": "~0.13.12"
  },
  "devDependencies": {
    "@babel/core": "^7.9.0",
    "@types/react": "~16.9.35",
    "@types/react-native": "~0.63.2",
    "typescript": "~4.0.0"
  },
  "private": true,
  "name": "xxx",
  "version": "1.0.0"
}
```

#### app.json

```json
{
  "expo": {
    "name": "xxx",
    "slug": "xxx",
    "version": "1.0.0",
    "orientation": "portrait",
    "icon": "./assets/icon.png",
    "splash": {
      "image": "./assets/splash.png",
      "resizeMode": "contain",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": ["**/*"],
    "ios": {
      "supportsTablet": true,
      "bundleIdentifier": "Weijland.IT.expo-firebase-starter"
    },
    "android": {
      "adaptiveIcon": {
        "foregroundImage": "./assets/adaptive-icon.png",
        "backgroundColor": "#FFFFFF"
      }
    },
    "web": {
      "favicon": "./assets/favicon.png"
    },
    "plugins": [
      "@react-native-firebase/app",
      "@react-native-firebase/perf",
      "@react-native-firebase/crashlytics"
    ]
  }
}

```

## Environment

<details><summary>Click To Expand</summary>
<p>

**Node --version: ** `v12.14.0`

**`react-native info` output:**

<!-- Please run `react-native info` on your terminal and paste the contents into the code block below -->

`react-native` is not installed.

</details>

--- a/packages/app/plugin/src/android/copyGoogleServices.ts
+++ b/packages/app/plugin/src/android/copyGoogleServices.ts
@@ -2,7 +2,7 @@ import { ConfigPlugin, withDangerousMod } from '@expo/config-plugins';
 
 import { DEFAULT_TARGET_PATH } from './constants';
 import path from 'path';
-import fs from 'fs/promises';
+import fs from 'fs';
 
 /**
  * Copy `google-services.json`
@@ -24,7 +24,7 @@ export const withCopyAndroidGoogleServices: ConfigPlugin = config => {
       const destPath = path.resolve(config.modRequest.platformProjectRoot, DEFAULT_TARGET_PATH);
 
       try {
-        await fs.copyFile(srcPath, destPath);
+        await fs.promises.copyFile(srcPath, destPath);
       } catch (e) {
         throw new Error(
           `Cannot copy google-services.json, because the file ${srcPath} doesn't exist. Please provide a valid path in \`app.json\`.`,

